client/Otiport-client/src/stories/screens/CreateUser/index.js

```diff
@@ -20,15 +22,20 @@ export default class CreateUser extends React.Component<Props, State> {
       firstName: "",
       birthDate: "",
       password: "",
-      middleName: ""
+      middleName: "",
+      listOfCountries: this.props.getCountries()
     };
   }
   handleChange = event => {
     this.setState({
       [event.target.id]: event.target.value
     });
   };
-  sendLogin = () => {
+  getCountries = () => {
+    //BU KISIMDA DATA CONSOLEDAN SONRA GELDIGI ICIN SORUN VAR.
```

Comment:
Burası reducer lar üzerinden geleceği için bu şekilde direkt method çağrılmamalı